azurelinuxagent/common/conf.py

```diff
@@ -85,6 +85,69 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
         raise AgentConfigError(("Failed to load conf file:{0}, {1}"
                                 "").format(conf_file_path, err))
 
+__SWITCH_OPTIONS__ = {
+    "OS.EnableRDMA" : False,
```

Comment:
Maybe we should use some sort of structure to contain these keys, which is referenced by the get methods for the individual values, to prevent missing any in the future? Perhaps enums or just a dictionary. Not critical now, we could improve later.